SPECS/kernel/config

```diff
@@ -115,20 +119,21 @@ CONFIG_BPF_JIT=y
 CONFIG_BPF_JIT_ALWAYS_ON=y
 CONFIG_BPF_JIT_DEFAULT_ON=y
 CONFIG_BPF_UNPRIV_DEFAULT_OFF=y
+CONFIG_USERMODE_DRIVER=y
 # CONFIG_BPF_PRELOAD is not set
-# CONFIG_BPF_LSM is not set
+CONFIG_BPF_LSM=y
 # end of BPF subsystem
 
-CONFIG_PREEMPT_NONE=y
-# CONFIG_PREEMPT_VOLUNTARY is not set
+# CONFIG_PREEMPT_NONE is not set
+CONFIG_PREEMPT_VOLUNTARY=y
```

Comment:
Another optimization where premtion elimates wait times for kernel https://stackoverflow.com/questions/10042550/disadvantages-of-preempt-rt#:~:text=CONFIG_PREEMPT_VOLUNTARY%20-%20This%20option%20introduces%20checks%20to%20the,degree%20but%20still%20it%20doesn%27t%20eliminate%20them%20totally.